compiler/src/yul/mappers/expressions.rs

```diff
@@ -38,11 +38,33 @@ pub fn expr(context: &Context, exp: &Spanned<fe::Expr>) -> Result<yul::Expressio
     }
 }
 
-pub fn expr_call(exp: &Spanned<fe::Expr>) -> Result<yul::Expression, CompileError> {
-    if let fe::Expr::Call { args: _, func } = &exp.node {
+pub fn expr_call(
+    context: &Context,
+    exp: &Spanned<fe::Expr>,
+) -> Result<yul::Expression, CompileError> {
+    if let fe::Expr::Call { args, func } = &exp.node {
         if let fe::Expr::Attribute { value: _, attr } = &func.node {
+            let arguments = &args.node;
+            let yul_args = arguments
+                .iter()
+                .map(|val| {
+                    if let fe::CallArg::Arg(arg_expr) = val.node.clone() {
+                        let spanned = Spanned {
+                            node: arg_expr,
+                            span: val.span,
+                        };
+                        return expr(context, &spanned);
+                    }
+                    unimplemented!()
+                })
+                .filter_map(Result::ok);
```

Comment:
we should just collect here

Reply:
Good point. I also think I was suppressing any errors with the `filter_map` and the `.collect::<Result<_, _>>()?;` throws them as it should be.